main/glossary/README.md

```diff
@@ -425,6 +425,22 @@ current allocation as a [payout](#payout).
 
 Records and tuples are immutable objects which can be passed by copy. They only contain values which are [passables](#passable). 
 For more details, see [Passable objects documentation](/guides/js-programming/far.md#rules-for-creating-passable-objects).
+
+## Remotable
+
+Remotables are objects intended to be used from other vats. Remote messages sent to remotables must only contain [passable](#passable)
+arguments and return passable results. 
+
+All of a remotable's property values must be functions, and cannot be accessors. Its methods are called by wrapping 
+the remotable object with [`E`](#e), such as `E(issuer).getBrand()`. As with all calls to `E()`, it results in a Promise.
+
+Every object returned from a smart contract, such a publicFacet or creatorFacet, must be passable. All objects used in 
+your contract's external API must be passable.
```

Comment:
I feel like there is a bit of repetition here, and slight confusion between remotable and presence.

Wondering if we could structure it the following way:
```suggestion
Remotables are objects containing only methods, intended to be used from other vats. The methods of remotables accept only [passable](#passable) arguments and return passable results. 

A remotable is used in other vats through its proxy called a [presence](#presence). A remotable's methods are called by wrapping the presence with [`E`](#e), such as `E(issuer).getBrand()`.
```

The last line feels more appropriate for the passable section.

@tyg, what do you think? Happy to include this proposed change when working on the other feedback.

Reply:
Since "Proxy" has a specific meaning in JS, and presences are not JS proxies, please avoid describing them using "proxy" terminology. "Representative" is good. (Our own internal use of "representative" for a different concept should not stop us. We can change our unexplained internal terminology if needed.)

Reply:
there are apparently other uses of the term proxy in relation to presence. I'll double check those.

Reply:
Actually, both the old text and the new text are not quite right on the Remotable/presence terminology. See https://github.com/Agoric/agoric-sdk/pull/2909/files . Warning: Pedantic clarification ahead. Almost certainly too much detail to actually include here. But it should constrain the text we do include here to be correct.

The thing with methods is a PrimaryRemotable. The remote presences are Remotes. Both PrimaryRemotables and Remotes are Remotables. At this level of abstraction, we talk about Remotes, but all Remotes are presences. Not all presences are Remotes though. It is the eventual-send level of abstraction that defines presences. Captp uses presences only to implement Remotes. However, other code is free to use eventual-send to create presences that are not Remotes.

Reply:
Attn @michaelfig , with whom we worked out the precise terminology.